app/liquid_tags/jsitor_tag.rb

```diff
@@ -0,0 +1,38 @@
+class JsitorTag < LiquidTagBase
+  PARTIAL = "liquids/jsitor".freeze
+  JSITOR_URL = /\A(https|http):\/\/(jsitor)\.(com)\/(embed)\/([a-zA-Z0-9]+)\Z/.freeze
+  def initialize(tag_name, link, token)
+    super
+    @link = jsitor_link_parser(link)
+  end
+
+  def render(_context)
+    ActionController::Base.new.render_to_string(
+      partial: PARTIAL,
+      locals: {
+        link: @link,
+        height: 600
+      },
+    )
+  end
+
+  private
+
+  def jsitor_link_parser(link)
+    parsed_link = ActionController::Base.helpers.strip_tags(link.strip)
+
+    return parsed_link if link_valid?(parsed_link)
+
+    jsitor_error
+  end
+
+  def link_valid?(link)
+    JSITOR_URL.match link
+  end
+
+  def jsitor_error
+    raise StandardError, "Invalid jsitor link"
```

Comment:
Suggestion - If we can add specific error by adding the example link then it will be more awesome. :-)

Reply:
also `JSitor` instead of `jsitor` :)